pysatNASA/instruments/templates/template_cdaweb_instrument.py

```diff
@@ -11,7 +11,8 @@
 
 Copy and modify this file as needed when adding a new Instrument to pysat.
 
-DO NOT include the NRL distribution statement in your new file.
+DO NOT include the NRL distribution statement in your new file. Statements for new
+files will have to be approved by NRL separately before the versioned release.
```

Comment:
👎

Reply:
I'm checking with a lawyer on these NRL statements.

Reply:
NRL does not have authority over pysat code which also means NRL does not get to decide which code to approve or disapprove from public release. I'd like to discuss a path forward in the meeting today.